packages/pds/src/lexicon/types/app/bsky/feed/embed.ts

```diff
@@ -5,7 +5,7 @@ import * as AppBskyActorRef from '../actor/ref'
 
 /** A list embeds in a post or document. */
 export interface Main {
-  items?: (Media | Record | External | Unknown)[];
+  items?: (Media | Record | External | { $type: string })[];
```

Comment:
Do we need to allow unknown keys on `{ $type: string }`?

Reply:
Yeah here's where I need to fix that.

Reply:
Fixed